docs/reference/commandline/swarm_leave.md

```diff
@@ -16,13 +16,16 @@ Usage:  docker swarm leave [OPTIONS]
 Leave a swarm
 
 Options:
-      --force   Force leave ignoring warnings.
+      --force   Force a node to leave the swarm, ignoring warnings.
       --help    Print usage
 ```
 
-This command causes the node to leave the swarm.
+This command causes the node to leave the swarm. When used on a manager,
+the swarm may lose quorum. To prevent this, join a new manager or promote an existing
+node to a manager. To avoid needing to use `--force` when removing a manager, demote
+the mananger to a worker using `docker node demote` before removing it from the swarm.
```

Comment:
I suggest saying upfront that `swarm leave` won't allow a manager to leave unless `--force` is specified. This is implied by "To avoid needing to use `--force`...", but I think it would be clearer to say it directly.

How about changing

> When used on a manager, the swarm may lose quorum. To prevent this, join a new manager or promote an existing node to a manager

to

> The `--force` option will allow a manager to leave the swarm, but this does not reconfigure the swarm to cleanly remove the node as a manager. This can put the swarm in a bad state where expects more managers than the actual number, which can cause it to lose quorum. The safe way to remove a manager from a swarm is to demote it to a worker and then have it leave the swarm without `--force`. `--force` should only be used in situations where the swarm will no longer be used after the manager leaves, such as a single-node swarm.